datadog/fwprovider/resource_datadog_integration_gcp_sts.go

```diff
@@ -267,6 +282,12 @@ func (r *integrationGcpStsResource) buildIntegrationGcpStsUpdateRequestBody(ctx
 	diags := diag.Diagnostics{}
 	attributes := datadogV2.NewGCPSTSServiceAccountAttributesWithDefaults()
 
+	accountTags := make([]string, 0)
+	if !state.AccountTags.IsNull() {
+		diags.Append(state.AccountTags.ElementsAs(ctx, &accountTags, false)...)
+	}
+	attributes.SetAccountTags(accountTags)
```

Comment:
Same comments apply here.

Reply:
Should we be doing something like the `IsUnknown()` for `AccountTags` (like we're doing below) since it's also a new field? Not familiar enough with terraform to know what's right here.